python/ray/remote_function.py

```diff
@@ -14,6 +14,8 @@
 DEFAULT_REMOTE_FUNCTION_CPUS = 1
 DEFAULT_REMOTE_FUNCTION_NUM_RETURN_VALS = 1
 DEFAULT_REMOTE_FUNCTION_MAX_CALLS = 0
+# Normal tasks may be retried on failure this many times.
+DEFAULT_REMOTE_FUNCTION_NUM_TASK_RETRIES = 3
```

Comment:
Could you please make this configurable? e.g. if there's a scenario that user doesn't want any task to be retried at all, it doesn't quite make sense for him to specify the `num_retries_allowed` as 0 for every single remote function, it would be a lot simpler to just change the default which take effects on all tasks.

Reply:
I think we can worry about how to do this later, it's a bit much for now.

Reply:
okay, could you please add a TODO to track this?